stdlib/streams/src/main/ballerina/src/streams/util/snapshot.bal

```diff
@@ -47,14 +47,22 @@ boolean stateLoaded = false;
 map<boolean> loadedStates = {};
 
 # Native function to deserialize a serialized snapshot.
+# This function is deprecated since v1.1.
```

Comment:
From my understanding the word `deprecate` in software world means that the function is no longer recommended (there are replaced with newer/better) functions and those will be removed in future. 

It seems like we are panicking an error with `not supported` message. 
Should we maybe completely remove this method without deprecating it? The word deprecate gives a false idea. 

WDYT?

Reply:
Thats a valid point, but these methods were publicly available in 1.0. We cannot simply remove them. If they are going to use these function, we have to anyway convey the user that functionality is not supported, hence the panic. :)

Reply:
In that case, shouldn't we keep the functions as it is in working state (marked as deprecated) and remove it in a major version?

Reply:
from stdlib point, java methods are not considered as API and as per further discussion, we decided to remove ballerina APIs and bump streams module version